CHANGELOG.adoc

```diff
@@ -67,6 +67,9 @@ The help text can include multiple lines (separated by `\n`), but no text format
 - https://github.com/eclipse-sirius/sirius-components/issues/2083[#2083] [diagram] Add support for edge markers
 - https://github.com/eclipse-sirius/sirius-components/issues/2086[#2086] [diagram] Add a panel with the default diagram actions
 - https://github.com/eclipse-sirius/sirius-components/issues/2087[#2087] [diagram] Add support for snap to grid
+- https://github.com/eclipse-sirius/sirius-components/issues/2115[#2115] [form] Add the possibility to choose the label position on checkbox widget.
+Note that the default position changes from top to end. +
+before : image:doc/screenshots/checkboxLabelPlacementBefore.png[checkboxLabelPlacementTop] after : image:doc/screenshots/checkboxLabelPlacementAfter.png[checkboxLabelPlacementEnd]
```

Comment:
The markup seems wrong; the preview does not show the images.
I think AsciiDoc expects the `image:` elments to be at the beginning of the line.
There are working examples elsewhere in this file.